src/flows/selectors/flowsSelectors.ts

```diff
@@ -11,6 +11,10 @@ export const selectNotebooks = (state: AppState): Notebook[] => {
 }
 
 export const selectShouldShowNotebooks = (state: AppState): boolean => {
+  if (isFlagEnabled('showNotebooksForCI')) {
+    return true
+  }
+
```

Comment:
TLDR version: If `showNotebooksForCI` is on, this selector should always return true.

--
Less TLDR version: 

This selector was producing issues in this e2e test because, when trying to force the display of notebooks for CI testing in Cloud, we would get this result:
1. `if (!CLOUD)` is `false` - skipped.
2. `if (!wasCreatedBeforeIOxCutoff & !isFlagEnabled('showNotebooksForCI'))` is false (the flag is on) - skipped.
3. Selector returns whether the number of notebooks in state is `> 0`.

3. is always false if the test hasn't set up a notebook yet. So we need to move the logic that forces the display of notebooks based on the `showNotebooksForCI` flag up, so that Cypress can navigate to the notebooks page to create whatever notebook it needs for the test.

Reply:
Or we could mock a notebook creation. See the `release.test`. 
https://github.com/influxdata/ui/blob/master/cypress/e2e/cloud/releases.test.ts#L165-L171

Reply:
👍